CMSgov/easi-app
# EASI-4053 Fix "Change Request Type" on System Intakes

# EASI-4053


## Changes and Description

There is a bug when changing an IT Gov Intake Request's type. Instead of updating the type property, a new intake is made.





## How to test this change

1. When creating a new  IT Gov System Intake Request, select **Add a new system or service** and continue
2. At the governance overview page, select **Change request type**
3. Select a any request type and continue
4. Check that an update query was used instead of create

Note: The path to Change request type was and still is only available thru the new system request type option
